pubsub-client/src/pubsub_client.rs

```diff
@@ -86,8 +86,8 @@
 //! # Ok::<(), anyhow::Error>(())
 //! ```
 
-pub use crate::nonblocking::pubsub_client::PubsubClientError;
```

Comment:
This is a breaking change to the public interface. It's okay to do to `solana-pubsub-client`, because that's not actually released yet (will be as of v1.16), but it also affects `solana-client`. You can either restore the `pub use`, or rexport PubsubClientError directly [here](https://github.com/solana-labs/solana/blob/db4b76d9e25dc119ca94a751d76079a34bfb8eb8/client/src/lib.rs#L28) and [here](https://github.com/solana-labs/solana/blob/db4b76d9e25dc119ca94a751d76079a34bfb8eb8/client/src/nonblocking/mod.rs#L13). The latter would be preferable, but less easy.

Reply:
i moved the enum back, that's easier